### DISCUSSION TITLE
RocketFuel Theme 1.1.0 released

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#2107] [Show and tell]
RocketFuel Theme has been updated 

Overview can been seen here https://studio-elf.net/oqtane/rocketfuel

**Comment by patrickjlee** [comment]
Unfortunately using the Rocket Fuel theme seems to have broken the Control Panel in Oqtane 6 for me: clicking on the links (e.g. Site Management) now doesn't do anything?

**Reply by leigh-pointer** [reply]
@patrickjlee please beware that RocketFe is an interactive Theme so you will need to set the site settings render mode to Interactive.

**Reply by patrickjlee** [reply]
Thanks - how do I do that please (especially as I can't get to Site Management or Site Settings)?  I also suggest that you add a warning to users when they install the theme that they will need to set the render mode to Interactive with a link to how to do this.

**Reply by leigh-pointer** [reply]
@patrickjlee to change the Site Settings render mode, the easiest way would be to update the the "RenderMode" field on the "Site" table to "Interactive".  Restarting the framework should start it now in Interactive mode.

You idea about the message is a really good point and I will action that, thank you.

**Reply by patrickjlee** [reply]
Thanks @leigh-pointer, that solved the problem!
